src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java

```diff
@@ -574,8 +591,8 @@ private void deleteDeployment(String project, String deployment) throws IOExcept
             client.services().inNamespace(project).withLabel(KubernetesLabels.COMPONENT_LABEL, deployment).list()
                     .getItems().forEach(service -> client.services().withName(service.getMetadata().getName())
                             .withPropagationPolicy(DeletionPropagation.BACKGROUND).delete());
-            if (client.isAdaptable(OpenShiftClient.class)) {
-                OpenShiftClient oclient = client.adapt(OpenShiftClient.class);
+            OpenShiftClient oclient = toOpenShiftClient(client);
+            if (oclient != null) {
```

Comment:
@sbouchet: the runtime behavior when adapting a kubernetes client to an openshift client changed in 6.x. I thus changed the code. I'd love you to test this too.